src/modules/triggers/exports.bicep

```diff
@@ -0,0 +1,57 @@
+// Source: 
+// Date: 2023-02-02
+// Version: 
+
+@description('Required. The name of the parent Azure Data Factory. Required if the template is used in a standalone deployment.')
+param dataFactoryName string
+
+@description('Required. The ID of the storage account.')
+param storageAccountId string
+
+var PipelineName = 'extract_parquet'
+var BlobContainerName = 'ms-cm-exports'
+var triggerName = 'ms_cm_exports_trigger'
+
+resource dataFactoryRef 'Microsoft.DataFactory/factories@2018-06-01' existing = {
+  name: dataFactoryName
+}
+
+resource trigger 'Microsoft.DataFactory/factories/triggers@2018-06-01' =  {
+  name: triggerName
+  parent: dataFactoryRef
+  properties: {
+    annotations: []
+    pipelines: [
+      {
+        pipelineReference: {
+          referenceName: PipelineName
+          type: 'PipelineReference'
+        }
+        parameters: {
+          folderName: '@triggerBody().folderPath'
+          fileName: '@triggerBody().fileName'
+        }
+      }
+    ]
+    type: 'BlobEventsTrigger'
+    typeProperties: {
+      blobPathBeginsWith: '/${BlobContainerName}/blobs/'
```

Comment:
Is `/blobs/` correct? Files don't get exported into that path.

Reply:
Yes.  The true path is https://uri/containername/blobs/foldername.  If you don't add the /blobs/ part ADF can't browse the storage.